components/monitoring/prometheus/base/monitoringstack/monitoringstack.yaml

```diff
@@ -114,9 +115,12 @@ spec:
         - '{__name__="controller_runtime_reconcile_errors_total", namespace!~".*-tenant|openshift-.*|kube-.*"}'
         - '{__name__="controller_runtime_reconcile_total", namespace!~".*-tenant|openshift-.*|kube-.*"}'
         - '{__name__="kube_pod_status_unschedulable", namespace!~".*-tenant|openshift-.*|kube-.*"}'
-        - '{__name__="kube_pod_container_status_restarts_total", namespace="openshift-pipelines"}'
-        - '{__name__="kube_pod_container_status_waiting_reason", namespace!~".*-tenant|openshift-.*|kube-.*"}'
-        - '{__name__="kube_pod_status_phase", namespace!~".*-tenant|openshift-.*|kube-.*"}'
```

Comment:
@theflockers  The changes introduced to those lines, exclude namespace `release-service` from the result that being forwarded for those metrics. Is that the intention?